Sources/CodeEditTextView/STTextViewController.swift

```diff
@@ -102,6 +107,8 @@ public class STTextViewController: NSViewController, STTextViewDelegate, ThemeAt
         rulerView.drawSeparator = false
         rulerView.baselineOffset = baselineOffset
         rulerView.font = NSFont.monospacedDigitSystemFont(ofSize: 9.5, weight: .regular)
+        rulerView.isHidden = !lineNumbers
```

Comment:
worth noting that this change is within `loadView()` to provide correct initial state, and is also handled in `reloadUI()`